init.go

```diff
@@ -19,25 +18,23 @@ func init() {
 		runtime.GOMAXPROCS(1)
 		runtime.LockOSThread()
 
+		// Configure logrus to talk to the parent.
 		level := os.Getenv("_LIBCONTAINER_LOGLEVEL")
 		logLevel, err := logrus.ParseLevel(level)
 		if err != nil {
 			panic(fmt.Sprintf("libcontainer: failed to parse log level: %q: %v", level, err))
 		}
+		logrus.SetLevel(logLevel)
 
 		logPipeFdStr := os.Getenv("_LIBCONTAINER_LOGPIPE")
 		logPipeFd, err := strconv.Atoi(logPipeFdStr)
 		if err != nil {
 			panic(fmt.Sprintf("libcontainer: failed to convert environment variable _LIBCONTAINER_LOGPIPE=%s to int: %s", logPipeFdStr, err))
 		}
-		err = logs.ConfigureLogging(logs.Config{
```

Comment:
It actually makes things worse because
- its two users have different needs;
- its existence makes the intention of libcontainer/logs package blurry (since the removal it's now clear that this it's about forwarding logs from runc init to the parent process).